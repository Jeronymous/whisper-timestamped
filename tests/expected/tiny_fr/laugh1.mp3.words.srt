1
00:00:00,060 --> 00:00:00,680
Je

2
00:00:00,680 --> 00:00:00,780
ne

3
00:00:00,780 --> 00:00:00,960
suis

4
00:00:00,960 --> 00:00:01,120
pas

5
00:00:01,120 --> 00:00:01,280
en

6
00:00:01,280 --> 00:00:01,380
train

7
00:00:01,380 --> 00:00:01,560
de

8
00:00:01,560 --> 00:00:01,660
faire

9
00:00:01,660 --> 00:00:01,760
Ã§a.

