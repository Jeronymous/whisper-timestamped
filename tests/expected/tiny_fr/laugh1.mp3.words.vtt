WEBVTT

00:00.060 --> 00:00.680
Je

00:00.680 --> 00:00.780
ne

00:00.780 --> 00:00.960
suis

00:00.960 --> 00:01.120
pas

00:01.120 --> 00:01.280
en

00:01.280 --> 00:01.380
train

00:01.380 --> 00:01.560
de

00:01.560 --> 00:01.660
faire

00:01.660 --> 00:01.760
Ã§a.

