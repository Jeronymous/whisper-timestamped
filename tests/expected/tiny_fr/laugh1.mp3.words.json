{
  "text": " Je ne suis pas en train de faire ça.",
  "segments": [
    {
      "id": 0,
      "seek": 0,
      "start": 0.06,
      "end": 1.76,
      "text": " Je ne suis pas en train de faire ça.",
      "tokens": [
        50364,
        2588,
        408,
        7624,
        1736,
        465,
        3847,
        368,
        4865,
        2788,
        13,
        50464
      ],
      "temperature": 0.0,
      "avg_logprob": -1.531521723820613,
      "compression_ratio": 0.8222222222222222,
      "no_speech_prob": 0.3798484802246094,
      "words": [
        {
          "text": "Je",
          "start": 0.06,
          "end": 0.68
        },
        {
          "text": "ne",
          "start": 0.68,
          "end": 0.78
        },
        {
          "text": "suis",
          "start": 0.78,
          "end": 0.96
        },
        {
          "text": "pas",
          "start": 0.96,
          "end": 1.12
        },
        {
          "text": "en",
          "start": 1.12,
          "end": 1.28
        },
        {
          "text": "train",
          "start": 1.28,
          "end": 1.38
        },
        {
          "text": "de",
          "start": 1.38,
          "end": 1.56
        },
        {
          "text": "faire",
          "start": 1.56,
          "end": 1.66
        },
        {
          "text": "ça.",
          "start": 1.66,
          "end": 1.76
        }
      ]
    }
  ],
  "language": "fr"
}